core/commonMain/src/kotlinx/serialization/descriptors/SerialDescriptor.kt

```diff
@@ -275,19 +292,17 @@ public interface SerialDescriptor {
      *     val e: List<Int> = listOf(1), // Optional == true
      * )
      * ```
-     * Returns `false` for valid indices of collections, maps and enums.
+     * Returns `false` for valid indices of collections, maps, and enums.
      *
      * @throws IndexOutOfBoundsException for an illegal [index] values.
      * @throws IllegalStateException if the current descriptor does not support children elements (e.g. is a primitive).
      */
-    @ExperimentalSerializationApi
     public fun isElementOptional(index: Int): Boolean
 }
 
 /**
  * Returns an iterable of all descriptor [elements][SerialDescriptor.getElementDescriptor].
  */
-@ExperimentalSerializationApi
 public val SerialDescriptor.elementDescriptors: Iterable<SerialDescriptor>
```

Comment:
While we are here, let's add a short sample of code?

Reply:
The same goes for all the other "stable" parts

Reply:
I've added examples for `SerialDescriptor` and its builders. I believe that `SerialKind`s don't have to have them